cpu/stm32_common/periph/flashpage.c

```diff
@@ -77,7 +76,8 @@ static void _erase_page(void *page_addr)
     uint32_t *dst = page_addr;
 #else
     uint16_t *dst = page_addr;
-
+#endif
+#if defined(CPU_FAM_STM32F0) || defined(CPU_FAM_STM32F1)
```

Comment:
This change should be in it's own commit and not in b72a6edd87b6a438694beb43178c4f04347f824a where it's unrelated.

Reply:
Why? Only `STM32F0 STM32F1` need to initialize HSI clock (at least for the implemented boards). My goal was to separate it from the default case which only said   ` /* Default is to support half-word sizes */`, But I guess you are right, the commit message is not explicit enough. Would you be Ok with just re-wording?

Reply:
No, this changes the initial behaviour as you say, so it needs to be in its own commit.